.gitignore

```diff
@@ -1,5 +1,8 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
+*/.DS_Store
+*.idea/*
```

Comment:
Can you add this to your global .gitignore?

Reply:
I didn't know global .gitignores existed. Sure